Tasks/Common/webdeployment-common/msdeployutility.ts

```diff
@@ -214,21 +237,28 @@ function getMSDeployInstallPath(registryKey: string): Q.Promise<string> {
  * error stream ( saved in error.txt) , display error to console
  * 2. Checks if there is file in use error , suggest to try app offline.
  */
-export function redirectMSDeployErrorToConsole() {
+export function redirectMSDeployErrorToConsole(publishProfile: any) {
     var msDeployErrorFilePath = tl.getVariable('System.DefaultWorkingDirectory') + '\\error.txt';
     
     if(tl.exist(msDeployErrorFilePath)) {
         var errorFileContent = fs.readFileSync(msDeployErrorFilePath);
```

Comment:
Can we call the `toString()` here ? and remove it from **errorFileContent.toString()** in all `if` conditions ?